examples/Common/Common.csproj

```diff
@@ -5,11 +5,9 @@
     <TargetFramework>net6.0</TargetFramework>
```

Comment:
net7 isn’t long term support
https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

I do think that users normally use LTS version. Since we are a client SDK our users would have to use the same versions as us, and it would give conflicts if we are 7 and they normally are 6.

What are you experience when using SDK’s?

Reply:
That's fair enough. Let's wait till November and update to version 8 👍